R/http.R

```diff
@@ -150,7 +150,7 @@ handleResponse <- function(response, error_call = caller_env()) {
     if (response$status %in% 200:399) {
       out <- json
     } else if (!is.null(json$error)) {
-      reportError(json$error)
+      reportError(unlist(json$error))
```

Comment:
From Connect, the `json$error` field is a simple string. 

```json
{
  "code": 123,
  "error": "something happened"
}
```

Reply:
I have also confirmed that shinyapps.io errors have a simple string for `json$string`.